src/controllers/dashboard/plugins/available/index.html

```diff
@@ -1,6 +1,10 @@
 <div id="pluginCatalogPage" data-role="page" class="page type-interior pluginConfigurationPage withTabs fullWidthContent">
     <div>
         <div class="content-primary">
+            <div class="inputContainer">
+                <label class="inputLabel" for="txtSearchPlugins">${Search}</label>
+                <input id="txtSearchPlugins" name="txtSearchPlugins" type="text" is="emby-input" class="emby-input">
```

Comment:
```suggestion
                <input id="txtSearchPlugins" name="txtSearchPlugins" type="text" is="emby-input" label="${Search}" />
```

The label element and class are added by the webcomponent. Having the label element created manually causes an error to be thrown in the console. Your other PR likely had the same issue, but I missed it when testing.

Reply:
Oops, you're right. 
Should I open a new PR to fix the other PR, or should change it in this PR?

Reply:
Either way would have been fine since it's a small fix and somewhat related... but I'll probably go ahead and merge this one so make it a separate PR please. :slightly_smiling_face: